babyagi.py

```diff
@@ -171,6 +189,9 @@ def context_agent(query: str, n: int):
         this_task_id = int(task["task_id"])
         print("\033[93m\033[1m"+"\n*****TASK RESULT*****\n"+"\033[0m\033[0m")
         print(result)
+        
+        # Step OannesGPT Addition  - Attempt to write output to .txt file
+        write_result_to_file("output_trail.txt", result)
```

Comment:
Is this better than running and outputting to file? E.g., `python babyagi.py > output_trail.txt`?

if so, this should only be done if there is a commandline arg to do it. 

Can you say more about the use case?

Reply:
Use case is to properly study how well data is being remembered and processed by the bot. I use it to generate python scripts and I want to see if he adds for example every script he manages to write he put up in his tasklist